src/algebra/group/to_additive.lean

```diff
@@ -80,25 +80,37 @@ optional doc string. -/
 @[derive has_reflect, derive inhabited]
 structure value_type : Type := (tgt : name) (doc : option string)
 
+meta def add_comm_prefix : bool → string → string
```

Comment:
```suggestion
/-- `add_comm_prefix x s` returns `"comm_" ++ s` if `x = tt` and `s` otherwise. -/
meta def add_comm_prefix : bool → string → string
```